src/Routes/NavRoutes.jsx

```diff
@@ -13,6 +16,9 @@ const NavRoutes = () => {
       <Route path="/products" element={<Products />} />
       <Route path="/cart" element={<Cart />} />
       <Route path="/wishlist" element={<Wishlist />} />
```

Comment:
Maybe you can use private route here for user specific routes: 
```js
<Route
            path="your-route"
            element={
              <RequireAuth>
                <YourComponent/>
              </RequireAuth>
            }
          />
```

Reply:
Yes, I will implement this while making Cart and Wishlist, in their respective PRs. Thank you.